### PR TITLE
Create documentation for `PhysicalConstants`.

### PR DESCRIPTION
**Pull request by simopier**
(Close. #91)

### CODE REVIEW

**Comment by simopier** [comment]
I have a questions for you, @cticenhour. I'm trying to add documentation for `PhysicalConstants.h`, which sits in utils, and I cannot find it when I create the documentation. 

it is possible to find it when typing [https://mooseframework.inl.gov/tmap8/docs/PRs/92/site/source/utils/PhysicalConstants.html](https://mooseframework.inl.gov/tmap8/docs/PRs/92/site/source/utils/PhysicalConstants.html), but it's not listed in the complete code syntax. 

I'm clearly missing something to include it to the syntax, but I cannot figure out what.

**Comment by cticenhour** [comment]
PhysicalConstants is not a registered MOOSE object. So it won't how up in syntax, and you'll have to create a link.

File: doc/bibliography/bib.bib
Comment (cticenhour) [inline doc/bibliography/bib.bib:1]: Without having my computer in front of me, consider placing this in a file named `doc/bib/tmap8.bib`. Without changing the docs configuration for TMAP8, MooseDocs will look in `doc/bib` first, I believe.

**Comment by simopier** [comment]
> PhysicalConstants is not a registered MOOSE object. So it won't show up in syntax, and you'll have to create a link on one of the other documentation pages.

How can I register a namespace?

**Comment by cticenhour** [comment]
> How can I register a namespace?

We do not register namespaces, only MOOSE objects for input files, underlying actions, and systems. I would say the best place to put this is in the heading content at the top of the syntax markdown file. I can explore fixes for this and push a change to this effect for your review.

**Comment by cticenhour** [comment]
@simopier Submitted two commits that I am considering part of the "review suggestions" portion of this, just committed pre-emptively. d1649f2 gets the documentation building - the file needed to be moved within the content directory, and I made a suggestion on a name change for it. 46a0fcb is a suggestion related to how to show the documentation link on both syntax pages. 

If you want to make further changes to it (or remove the latter commit altogether), feel free. I can also make modifications, or remove commits entirely.

File: doc/bibliography/bib.bib
Comment (cticenhour) [reply doc/bibliography/bib.bib:1]: Resolved with d1649f2.

**Comment by simopier** [comment]
@cticenhour , thank you very much! That looks perfect to me!

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
Taking that as approval of my commits, and I formally approve the rest of the changes!

**Pull request merged**